### PR TITLE
parser: Split parser test cases

### PR DESCRIPTION
**Pull request by shenli**
Avoid pr conflict

### CODE REVIEW

File: parser/parser_test.go [hunk 1]
```diff
@@ -49,26 +49,90 @@ func (s *testParserSuite) TestOriginText(c *C) {
 		FROM stuff`)
 }
 
-// TODO: table 43 and 50 parse failed
 func (s *testParserSuite) TestParser0(c *C) {
```
Comment (siddontang) [inline parser/parser_test.go]: TestSimple may be better here.

Comment (shenli) [inline parser/parser_test.go]: PTAL

**Comment by siddontang** [comment]
LGTM

File: parser/parser_test.go [hunk 2]
```diff
@@ -49,26 +49,90 @@ func (s *testParserSuite) TestOriginText(c *C) {
...
 }
 
-// TODO: table 43 and 50 parse failed
-func (s *testParserSuite) TestParser0(c *C) {
-	table := []struct {
-		src string
-		ok  bool
-	}{
+func (s *testParserSuite) TestSimple(c *C) {
+
```
Comment (qiuyesuifeng) [inline parser/parser_test.go]: Remove empty line?

Comment (shenli) [inline parser/parser_test.go]: PTAL

**Pull request merged**